frontend/tests/components/homepage/__snapshots__/HomepageExperimental.test.tsx.snap

```diff
@@ -0,0 +1,132 @@
+          <p
+            class="font-sans-md line-height-sans-4"
+          >
+            Search real Grants.gov data here on Simpler.Grants.gov. Our search aims to deliver closer matches to your keywords and filters. We also redesigned the results and opportunity listings to make them easier to navigate and read.
+          </p>
+          <a
+            href="/search"
+          >
+            <button
+              class="usa-button usa-button--big margin-y-2 usa-button--secondary"
+              data-testid="button"
+              type="button"
+            >
+              Try the new simpler search
+            </button>
+          </a>
+          <h3>
+            What you can't do quite yet
+          </h3>
+          <p
+            class="font-sans-md line-height-sans-4"
+          >
+            For now, you need to visit Grants.gov to access more advanced features like applying. We're working to bring the application process to this website soon. In the meantime, follow our roadmap to stay updated on our progress.
+          </p>
+          <div
+            class="grid-row grid-gap-md padding-y-2"
+            data-testid="grid"
+          >
+            <div
+              class="grid-col-6"
+              data-testid="grid"
+            >
+              <svg
+                aria-hidden="true"
+                class="usa-icon usa-icon--size-4 text-middle"
+                role="img"
+              >
+                <use
+                  href="/img.jpg#build"
+                />
+              </svg>
+              <h3>
+                Send us your feedback and suggestions.
+              </h3>
+              <p
+                class="font-sans-md line-height-sans-4"
+              >
+                We're excited to hear from you to learn how we can improve.
+              </p>
+              <a
+                class="font-sans-md line-height-sans-4"
+                href="mailto:simpler@grants.gov"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                Contact us at simpler@grants.gov
+              </a>
+            </div>
+            <div
+              class="grid-col-6"
+              data-testid="grid"
+            >
+              <svg
+                aria-hidden="true"
+                class="usa-icon usa-icon--size-4 text-middle"
+                role="img"
+              >
+                <use
+                  href="/img.jpg#mail"
+                />
+              </svg>
+              <h3>
+                Be the first to hear about new features.
+              </h3>
+              <p
+                class="font-sans-md line-height-sans-4"
+              >
+                Our newsletter delivers the latest news straight to your inbox.
+              </p>
+              <a
+                class="font-sans-md line-height-sans-4"
+                href="/subscribe"
+                rel="noopener noreferrer"
+                target="_blank"
```

Comment:
Michelle decided we don't want this in a new tab. Can you make that change?

Reply:
updated!